HTTP_HEADER_FORMAT.md

```diff
@@ -27,41 +26,41 @@ Is a 1-byte representing a 8-bit unsigned integer. Version 255 reserved.
 
 #### Format
 
-`base16(<trace-id>)-base16(<span-id>)-base16(<trace-options>)`
+```base16(<trace-id>)-base16(<span-id>)[-base16(<trace-options>)]```
 
-All fields are required. Character `-` is used as a delimiter between fields.
+```trace-id``` and ```span-id``` are required. The ```trace-options``` is optional. Character `-`
+ is used as a delimiter between fields.
 
 #### Trace-id
 
-Is the ID of the whole trace forest. It is represented as a 16-bytes array,
-e.g., `4bf92f3577b34da6a3ce929d0e0e4736`. All bytes 0 is considered invalid.
+Is the ID of the whole trace forest. It is represented as a 16-bytes array, e.g., 
+```4bf92f3577b34da6a3ce929d0e0e4736```. All bytes 0 is considered invalid.
 
-Implementation may decide to completely ignore the trace-context if the trace-id
-is invalid.
+Implementation may decide to completely ignore the trace-context if the trace-id is invalid.
 
 #### Span-id
 
-Is the ID of the caller span (parent). It is represented as a 8-bytes array,
-e.g., `00f067aa0ba902b7`. All bytes 0 is considered invalid.
+Is the ID of the caller span (parent). It is represented as a 8-bytes array, e.g., 
+```00f067aa0ba902b7```. All bytes 0 is considered invalid.
 
-Implementation may decide to completely ignore the trace-context if the span-id
-is invalid.
+Implementation may decide to completely ignore the trace-context if the span-id is invalid.
 
 #### Trace-options
 
-Controls tracing options such as sampling, trace level etc. It is a 1-byte
-representing a 8-bit unsigned integer. The least significant bit provides
-recommendation whether the request should be traced or not (1 recommends the
-request should be traced, 0 means the caller does not make a decision to trace
-and the decision might be deferred). The flags are recommendations given by the
-caller rather than strict rules to follow for 3 reasons:
+Controls tracing options such as sampling, trace level etc. It is a 1-byte representing a 8-bit 
+unsigned integer. The flags are recommendations given by the caller rather than strict rules to 
+follow for 3 reasons:
 
-1.  Trust and abuse.
-2.  Bug in caller
-3.  Different load between caller service and callee service might force callee
-    to down sample.
+1. Trust and abuse.
```

Comment:
It was never clear to me what this means. Maybe a bit of an explanation?

Reply:
I believe this meant a bad actor can always say sampled for a ton of requests. If you don't trust the sampled bit, you can ignore it.